xdsl/utils/deprecation.py

```diff
@@ -25,4 +25,5 @@ def new_func(*args: _P.args, **kwargs: _P.kwargs) -> _T:
 
 
 def deprecated_constructor(func: Callable[_P, _T]) -> Callable[_P, _T]:
-    return deprecated(f'use the constructor (`ClassName(...)`) instead.')(func)
+    # TOFIX: improve printing
+    return deprecated(f'{"use the constructor (`ClassName(...)`) instead."}')(func)
```

Comment:
I'm not sure I understand this change? Why do we want to add the `"`?

Also, what do you mean by "improve printing" here? Do you have an idea on what we could do?

I would also say that this `TOFIX` should be moved as a GitHub issue, since I'm not sure that this bring value to the code here, if that makes sense?